source/glossary.rst

```diff
@@ -1,10 +1,11 @@
-.. _terminology:
-
-============
-Terminology
-============
+.. _glossary:
```

Comment:
There are 3 choices for references and glossaries

1. - shared
2. - component-level
3. - shared & component-level

I am OK with all 3 choices. In the discussion earlier this week, someone said shared glossary, which seemed like a good idea at the time, but now I am not sure. If you do not like it, we can drop it.

If we do (2), I think we could put some of common terms in config file so we don't duplicate the text.

Most people only look at glossary/references by following a link from the main document, so I don't think it is a problem that a glossary might be dominated by a component, or if we have shared and component-level glossaries.

Reply:
From my perspective, the common glossary or bibliography for terms/papers used  across components (e.g., sycl, dpc++, device,..., reference to some standards,..) makes sense. "Partitioner" is TBB specific and might be in the glossary of the component, feature vector - DAL specific and might be in the glossary of the component
@akukanov what is your perspective?

Reply:
For sure we need to have common part so there will be no diffident definitions on same terms. 
But on product specific content  and a fact that it will not be relevant for other products - i assume that primary usage mode for terminology would be ether direct links from documentation, or user will lookup for particular term.  So having large common terminology section shouldn't be a problem as usually people are not going to look through it?